web/src/app/users/UserDetails.js

```diff
@@ -62,6 +63,7 @@ export default class UserDetails extends React.PureComponent {
   state = {
     createCM: false,
     createNR: false,
+    verifyCMID: null,
```

Comment:
if we could rename this to `showVerifyDialogByID` to be more clear as to what it's being used for